sklearn/metrics/pairwise.py

```diff
@@ -412,8 +412,10 @@ def nan_euclidean_distances(
     Parameters
     ----------
     X : array-like of shape=(n_samples_X, n_features)
+        Array of vectors.
```

Comment:
We do not really have a consistent description for `X` or `y` in `pairwise.py`. I think we can adopt the one from `euclidean_distances`  and use it through out this file:

```suggestion
        An array where each row is a sample and each column is a feature.
```

For reference:

https://github.com/scikit-learn/scikit-learn/blob/1c24595c74e0bea246737b19f8fdfc8a1ffa2282/sklearn/metrics/pairwise.py#L251-L252

Reply:
Implemented! Will update other relevant PRs as well, thanks.